src/inmanta/server/config.py

```diff
@@ -175,15 +175,6 @@ def validate_fact_renew(value: object) -> int:
     "server", "fact-resource-block", 60, "Minimal time between subsequent requests for the same fact", is_time
 )
 
-server_autrecompile_wait = Option(
```

Comment:
I am wondering whether we should make the transition to the new setting a bit more smooth. I mean:
* When the global server setting is set, use it and log a deprecation message.
* When the global server setting is not set, use the environment setting.

We can then remove the global server config setting in ISO7.